app/presenters/report_presenter.rb

```diff
@@ -66,6 +60,16 @@ def percentage_for(number:, total:)
   end
 
   def completed_defects(priority:)
-    defects_by_priority(priority: priority).completed
+    defects_by_priority(priority: priority).completed.where.not(actual_completion_date: nil)
```

Comment:
Does this imply that the `completed` scope is wrong?

Reply:
It's actually a data issue, some older completed defects don't have an `actual_completion_date` - I'm going to run this solution past the Hackney team, as I'm unsure if they want us to somehow include these in the report. Might be worth changing the scope if we want to ignore them, you're right.

Reply:
This feels like a very hacky workaround...

Reply:
Yeah, I'm going to find out if we want to run a migration instead, but leaving this until I've spoken to Hackney